apps/institution/serializers.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import serializers
+from django.utils.translation import ugettext as _
+
+from apps.institution.models import Institution
+
+
+class InstitutionSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Institution
+        fields = '__all__'
+
+    def validate(self, data):
+        province = data['province']
+        district = data['district']
+        municipality = data['municipality']
```

Comment:
Let's only define municipality in the serializer and auto-fill district and province on the server-side.